src/ducks/transactions/actions/AttachedDocsAction/BillChip.jsx

```diff
@@ -71,7 +71,9 @@ export class DumbBillChip extends React.PureComponent {
               <Figure total={bill.amount} coloredPositive signed symbol="€" />
             </>
           ) : (
-            t('Transactions.actions.attachedDocs.bill')
+            <>
+              {t('Transactions.actions.attachedDocs.bill')} {vendorName}
```

Comment:
Seems the vendorName should be part of the translation string `Blabla %{vendorName}` since in some languages, the vendorName should be inside/upfront of the string.

Nit